### PR TITLE
Release notes 21.2.0

### PR DESCRIPTION
**Pull request by moishce**
## Status
In Progress

### CODE REVIEW

File: content-repo/extra-docs/releases/21.2.0.md [hunk 1]
```diff
@@ -0,0 +1,1585 @@
...
+
+### Agari Phishing Defense Pack v1.0.1 (Partner Supported)
+#### Classifiers
+##### Agari Phishing Defense - Mapper
+Removed non-existing incident fields.
+
+##### Agari Phishing Defense - Classifier
+Removed non-existing incident fields.
+
+#### Layouts
```
Comment (Itay4) [inline content-repo/extra-docs/releases/21.2.0.md:458]: either remove this or add the layout

File: content-repo/extra-docs/releases/21.2.0.md [hunk 2]
```diff
@@ -0,0 +1,1585 @@
...
+---
+
+### Ansible Tower Pack v1.0.1
+#### Integrations
+##### Ansible Tower
+Fixed an issue where some arguments were incorrectly marked as default arguments.
+
+---
+
+### ApiModules Pack v2.2.0
```
Comment (Itay4) [inline content-repo/extra-docs/releases/21.2.0.md:487]: remove this pack

File: content-repo/extra-docs/releases/21.2.0.md [hunk 3]
```diff
@@ -0,0 +1,1585 @@
...
+- **similarIncidents**
+- **App message**
+
+#### Indicator Fields
+##### Mitre Tactics
+**Download URL**
+
+---
+
+### ComputerVisionEngine Pack v1.0.1 (Community Contributed)
```
Comment (Itay4) [inline content-repo/extra-docs/releases/21.2.0.md:692]: merge this with the new release notes of this pack

File: content-repo/extra-docs/releases/21.2.0.md [hunk 4]
```diff
@@ -0,0 +1,1585 @@
...
+
+### CyberX - Central Manager Pack v1.0.1 (Community Contributed)
+#### Integrations
+##### CyberX - Central Manager
+- Fixed display names for various integration parameters.
+- Upgraded the Docker image to: *demisto/python3-deb:3.9.1.15758*.
+
+---
+
+### Cyberint Pack v1.0.1 (Partner Supported)
```
Comment (Itay4) [inline content-repo/extra-docs/releases/21.2.0.md:759]: merge this with the new release notes of this pack

File: content-repo/extra-docs/releases/21.2.0.md [hunk 5]
```diff
@@ -0,0 +1,1585 @@
...
+##### Proofpoint Protection Server (Deprecated)
+- Deprecated. Use Proofpoint Protection Server v2 instead.
+- Improved deprecation comment.
+
+##### New: Proofpoint Protection Server v2
+Proofpoint email security appliance.
+
+##### Proofpoint Protection Server v2
+- Fixed an issue where the *Trust any certificate* integration parameter was not handled properly.
+- Upgraded the Docker image to demisto/python3:3.9.1.15759.
```
Comment (Itay4) [inline content-repo/extra-docs/releases/21.2.0.md:1214]:
```suggestion
```

File: content-repo/extra-docs/releases/21.2.0.md [hunk 6]
```diff
@@ -0,0 +1,1585 @@
...
+
+##### New: User Profile - Salesforce (Outgoing)
+(Available from Cortex XSOAR 6.0.0).
+
+#### Integrations
+##### New: Salesforce IAM
+Integrate with Salesforce's services to perform Identity Lifecycle Management operations. (Available from Cortex XSOAR 6.0.0).
+
+##### Salesforce IAM
+Added the import for *IAMApiModule* to support all IAM Classes.
```
Comment (Itay4) [inline content-repo/extra-docs/releases/21.2.0.md:1286]:
```suggestion
```

File: content-repo/extra-docs/releases/21.2.0.md [hunk 7]
```diff
@@ -0,0 +1,1585 @@
...
+##### Sixgill DarkFeed Enrichment
+Fixed an issue where some arguments were incorrectly marked as default arguments.
+
+---
+
+### Skyformation Pack v1.0.2 (Partner Supported)
+#### Integrations
+##### Skyformation (Deprecated)
+- Deprecated. Following M&A, partner has declared end of life for this integration.
+- Documentation and metadata improvements.
```
Comment (Itay4) [inline content-repo/extra-docs/releases/21.2.0.md:1415]:
```suggestion
```

**Comment by ShirleyDenkberg** [comment]
@Itay4 Doc review completed.

File: content-repo/extra-docs/releases/21.2.0.md [hunk 8]
```diff
@@ -0,0 +1,1516 @@
...
+**Orca Alert**
+
+#### Integrations
+##### Orca
+Agentless, workload-deep, context-aware security and compliance for AWS, Azure, and GCP.
+
+
+---
+
+### New: Palo Alto Networks Automatic SLR (Community) Pack v1.0.0 (Community Contributed)
```
Comment (Itay4) [inline content-repo/extra-docs/releases/21.2.0.md:314]:
```suggestion
### New: Palo Alto Networks Automatic SLR Pack v1.0.0 (Community Contributed)
```

File: content-repo/extra-docs/releases/21.2.0.md [hunk 9]
```diff
@@ -0,0 +1,1516 @@
...
+#### Integrations
+##### Palo Alto Networks Automatic SLR (Community)
```
Comment (Itay4) [inline content-repo/extra-docs/releases/21.2.0.md:316]:
```suggestion
##### Palo Alto Networks Automatic SLR
```

**Review by Itay4 — APPROVED** [review]
please note my comments

**Pull request merged**